src/frontend-scripts/components/section-main/Gamechat.jsx

```diff
@@ -1083,7 +1083,7 @@ class Gamechat extends React.Component {
 				>
 					<h2 className="ui header">
 						DANGER. Leaving an in-progress game will ruin it for the other players (unless you've been executed). Do this only in the case of a game already
-						ruined by an AFK/disconnected player or if someone has already left.
+						ruined by an AFK/disconnected player, if someone has already left, or if the game has been remade.
```

Comment:
I'm not sure if this is necessary, if I remember correctly when isRemade is true it automatically doesn't show this modal on clicking leave game (will test prior to merge)

Reply:
IIRC it did show it, but you are probably right.

Reply:
You may be right, it's 2AM and I'm speaking from memory so :shrug:

Reply:
I just tested it and the modal does show up.